config/locales/models/groupings/en.yml

```diff
@@ -3,4 +3,12 @@ en:
   activerecord:
     attributes:
       grouping:
+        inactive_groups:
+          one: 1 inactive group
+          other: "%{count} inactive groups"
```

Comment:
The quotes are needed because the string starts with an interpolation

https://stackoverflow.com/a/43702790